src/pkcs11/framework-pkcs15.c

```diff
@@ -4605,7 +4608,7 @@ pkcs15_pubkey_get_attribute(struct sc_pkcs11_session *session, void *object, CK_
 	case CKA_MODIFIABLE:
 	case CKA_EXTRACTABLE:
 		check_attribute_buffer(attr, sizeof(CK_BBOOL));
-		*(CK_BBOOL*)attr->pValue = FALSE;
+		*(CK_BBOOL*)attr->pValue = TRUE;
```

Comment:
I don't think public keys should be `CKA_MODIFIABLE` by default without any override.

Reply:
Okay, You caught me being lazy ;-)  I'll address that in a rebased PR